docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.1-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=smallboard_user
+      - POSTGRES_PASSWORD=smallboard_password
+      - POSTGRES_DB=smallboard
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
```

Comment:
We should probably include the `manage.py migrate` command in here to ensure that this works out of the box. Otherwise, simply running `docker-compose up` on a fresh repo will fail due to unapplied migrations. The `migrate` command is idempotent, so it should be safe to just tack on here.

e.g.

```
command: sh -c "python manage.py migrate && python manage.py runserver ..."
```

Reply:
Yeah I was on the fence about this one, I think it might be a little unexpected if we ran migrations automatically locally and maybe could do something bad if you ran this in the middle of making a change. On the flipside that might not be a big deal. I think if we ended up wanting to deploy with this we would definitely want to make sure it ran though.

Reply:
Ok, decided to just do it (and run collectstatic as well)

Reply:
Ah just saw this, but yeah I agree with your assessment of the tradeoffs 👍 I guess for development, we can encourage just spinning up the docker compose once, then making changes and having the running django instance hot-reload changes as usual (assuming we've mounted the dev's local code into the container volume).